test/app.test.js

```diff
@@ -687,76 +688,27 @@ describe('app', function() {
       });
     });
 
-    it('accepts null dataSource', function() {
-      app.model('MyTestModel', { dataSource: null });
-    });
```

Comment:
Hmm, github formatted the patch in such way that I did not realise this test is preserved below. Ignore my comment.